broker/core/scheduler-jobs/src/jobs/MeterInstanceJob.js

```diff
@@ -190,8 +190,8 @@ class MeterInstanceJob extends BaseJob {
     }, {
       maxAttempts: 4,
       minDelay: 1000
-    })
-      .tap(response => logger.info('Successfully updated meter state : ', response));
+    });
+    // .tap(response => logger.info('Successfully updated meter state : ', response));
```

Comment:
Not sure why. Un commenting this line make a lot of test fail, with `RangeError: Maximum call stack size exceeded `

Reply:
Remove the http respose object from the result returned by apiserver client and it is fixed now.

Reply:
Was it because of multiple calls dumping huge response object then ?

Reply:
It was probably due to some cyclic linkage within the http respose object returned by k8s lib and winton was recurively going inside to printt, resulting in a infinite recursive call loop